### ISSUE TITLE
Ruby 2.7 Warning: Using the last argument as keyword parameters is deprecated

### ISSUE THREAD

**Issue by harmdewit** [newrelic/rpm#314]
In Ruby 2.7.0 I'm getting warnings for keyword/positional arguments separation. For example something like this:

```ruby
require 'new_relic/agent/method_tracer'
class An
  def self.process(a: , b:)
  end
  class << self
    include NewRelic::Agent::MethodTracer
    add_method_tracer :process
  end
end

An.process(a: 1, c: 2)
# => .gem/ruby/2.7.0/gems/newrelic_rpm-6.8.0.360/lib/new_relic/agent/method_tracer.rb:329: 
# warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

This is because the traced methods have `(*args, &block)` as argument definitions instead of `(*args, **kwargs, &block)`. Alternatively you could check for Ruby 2.7 and use the new `...` syntax like in the Rails `delegate` https://github.com/rails/rails/pull/38023.

**Comment by rachelleahklein** [comment]
Hi @harmdewit! I'm happy to inform you that full Ruby 2.7.0 support will be released with our next agent, which will eliminate the warnings you are seeing.

Thanks for reaching out!

**Comment by tannalynn** [comment]
Hello! This has been fixed in agent version 6.9.0.
